src/config.ps1

```diff
@@ -0,0 +1,95 @@
+function ConvertTo-HashTable {
+	param (
+		[Parameter(ValueFromPipeline)]
+		[PSCustomObject]$Object
+	)
+	$Table = @{}
+	$Object.PSObject.Properties | ForEach-Object {
+		$V = $_.Value
+		if ($V -is [Array]) {
+			$V = [System.Collections.ArrayList]$V
+		} elseif ($V -is [PSCustomObject]) {
+			$V = ($V | ConvertTo-HashTable)
+		}
+		$Table.($_.Name) = $V
+	}
+	return $Table
+}
+
+function GetPwrHome {
+	if ($PwrHome) {
+		$PwrHome
+	} elseif ($env:PwrHome) {
+		$env:PwrHome
+	} else {
+		"$env:LocalAppData\pwr"
+	}
+}
+
+function GetPwrPullPolicy {
+	if ($PwrPullPolicy) {
+		$PwrPullPolicy
+	} elseif ($env:PwrPullPolicy) {
+		$env:PwrPullPolicy
+	} else {
+		"IfNotPresent"
+	}
+}
+
+function GetPwrDBPath {
+	"$(GetPwrHome)\pwrdb"
```

Comment:
Same sorta question as above, why not just name this `$(GetPwrHome)\db`?

Reply:
ok

Reply:
Done